textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/neextrator/NamedEntityExtractor.java

```diff
@@ -16,34 +18,30 @@
 import edu.uci.ics.textdb.common.field.Span;
 import edu.uci.ics.textdb.common.utils.Utils;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Properties;
-
-
 /**
  * @author Feng [sam0227] on 4/27/16.
  *         <p>
- *         Wrap the Stanford NLP Named Entity Recognizer as an operator.
- *         This operator would recognize 7 classes: Location, Person, Organization, Money, Percent, Date and Time.
- *         Return the recoginized data as a list of spans that are appended to the original tuple as a field.
+ *         Wrap the Stanford NLP Named Entity Recognizer as an operator. This
```

Comment:
Notice that @sam0227 has renamed this class.  So make sure to merge the master into your branch @rajesh9625 .

Also a general comment: it's a good idea to polish the code.  Meanwhile, to make the PR review easier, it will be good to do reformatting/polishing as a separate PR.